install_lightbits.sh

```diff
@@ -109,7 +115,7 @@ DisplayHelp()
     p    Password - use SINGLE quotes ''.        'p@ssword12345!!'
     k    Path to key.                            /home/root/keys/key.pem
     t    Lightbits Repository token.             QWCEWVDASADSSsSD
-    v    Lightbits Version.                      lightos-3-5-1-rhl-8, lightos-3-6-1-rhl-8, lightos-3-7-1-rhl-8, lightos-3-7-1-rhl-9
+    v    Lightbits Version.                      lightos-3-5-1-rhl-8, lightos-3-6-1-rhl-8, lightos-3-7-1-rhl-8, lightos-3-7-1-rhl-9, lightos-3-8-1-rhl-8
```

Comment:
@husam-lb if you could also add 3-9-1 that would be great

Reply:
3-9-1? 
do you mean lightos-3-6-1-rhl-9? I don't know the build number